app/build.gradle

```diff
@@ -67,24 +67,34 @@ android {
     }
 }
 
+apply plugin: 'maven'
+def dexmakerVersion = '17cc9cc'
+def dexmakerGroupId = 'com.github.crittercism.dexmaker'
+
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
 
     testCompile 'junit:junit:4.12'
+    // mockito
+    testCompile 'org.mockito:mockito-core:1.10.19'
 
+    // android test libraries
     androidTestCompile 'com.android.support.test:runner:0.5'
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
 
+    // mockito
+    androidTestCompile 'org.mockito:mockito-core:1.10.19'
+    androidTestCompile "${dexmakerGroupId}:dexmaker:${dexmakerVersion}"
```

Comment:
I'm sorry and I really apologise that I changed my mind,
that it would be rather good not to use ``initMocks`` than to depend on non-release version, I think.
But If you really want to use that method, I won't complaint it.
In that case you should add a note why we are using this version and delete three commented-out lines.